site/content/extend/plugins/best-practices.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Example Plugins"
+subsection: Plugins (Beta)
+weight: 50
+---
+
+See here for [server-specific best practices for plugins](/extend/plugins/server/best-practices/). Webapp-specific best practices are incoming.
+
+## How can I review the entire code base of a plugin?
+
+Carry out the following steps:
+
+1. Take a backup of project directory
+2. Create a dummy-master branch with no code:
+
+   ```
+   git checkout --orphan dummy-master #create the dummy master branch
```

Comment:
2/5 to not add comments in the same line as commands. This way it's not easy copy&past-able.